packages/bpk-docs/src/pages/ModalsPage/ModalsPage.js

```diff
@@ -146,6 +146,14 @@ const ModalsPage = () => (
     ]}
     components={components}
     readme={modalReadme}
+    usageTable={{
+      dos: ['Use for rich / complex content related to the current page.'],
```

Comment:
What is "rich / complex content"? Do we have an example of the opposite? These do's and don't should be really simple and obvious.

Reply:
Maybe something along the lines of "complex content, e.g. forms, multiple controls"?